src/polly2scc.py

```diff
@@ -1193,60 +1243,121 @@ def __read_picasso_results(self, filename, *args):
         }
 
         # capsule data into 355 data container
-        if 'aerBsc_raman_355' in pData.keys():
-            smoothWin_355 = self.picasso_attri_parser(
-                pData['aerBsc_raman_355'].retrieving_info,
-                varname='smoothing_window')
-            refH_bottom_355 = self.picasso_attri_parser(
-                pData['aerBsc_raman_355'].retrieving_info,
-                varname='reference_search_bottom')
-            refH_top_355 = self.picasso_attri_parser(
-                pData['aerBsc_raman_355'].retrieving_info,
-                varname='reference_search_top')
-            angstr_355 = self.picasso_attri_parser(
-                pData['aerBsc_raman_355'].retrieving_info,
-                varname='angstroem_exponent')
-            refVal_355 = self.picasso_attri_parser(
-                pData['aerBsc_raman_355'].retrieving_info,
-                varname='reference_value')
-
-            # calculate the backscatter-ratio at the reference height
-            refMask355 = (pData['height'][:] >= refH_bottom_355) & \
-                         (pData['height'][:] <= refH_top_355)
-            refBscMol355 = np.nanmean(
-                beta_pi_rayleigh(
-                    355,
-                    pressure=np.float64(pData['pressure'][refMask355]),
```

Comment:
See here. Conversion from `np.float64` to `np.float32` is gone. (+ the line below)